gef.py

```diff
@@ -5379,15 +5388,15 @@ class PCustomListCommand(PCustomCommand):
     _cmdline_ = "pcustom list"
     _syntax_ = "{:s}".format(_cmdline_)
 
-    def __init__(self):
+    def __init__(self) -> None:
         super().__init__()
         return
 
-    def do_invoke(self, argv):
+    def do_invoke(self, argv: List) -> None:
```

Comment:
An an aside, we should probably be parsing args into a tuple, not a list. Can you please add that to the project?

Reply:
done. BTW you can also add things to the linked project (having said that I don't have a problem doing it)